l10n_ar_ux/controllers/portal.py

```diff
@@ -34,22 +34,35 @@ def details_form_validate(self, data):
 
     @route()
     def account(self, redirect=None, **post):
-        if post:
-            error, _error_message = self.details_form_validate(post)
-            if not error:
-                post.pop('commercial_partner_id', False)
-                post.pop('vat', False)
-                post.pop('l10n_latam_identification_type_id', False)
-                post.pop('l10n_ar_afip_responsibility_type_id', False)
+        # if post:
+        #     error, _error_message = self.details_form_validate(post)
+        #     if not error:
+        #         post.pop('commercial_partner_id', False)
+        #         post.pop('vat', False)
+        #         post.pop('l10n_latam_identification_type_id', False)
+        #         post.pop('l10n_ar_afip_responsibility_type_id', False)
 
```

Comment:
aca, antes de llamar al super, hacer  llamada al nuevo values_preprocess_ar asi vemos que los campos esten bien y queden storeados. solo hacerlo en caso que hay valores por escribir

```suggestion
   if post and request.httprequest.method == 'POST':
         post = self.values_preprocess_ar(post)
```